common-lib/src/main/scala/com/gu/mediaservice/lib/aws/S3.scala

```diff
@@ -29,10 +30,19 @@ class S3(credentials: AWSCredentials) {
   lazy val client: AmazonS3 =
     new AmazonS3Client(credentials) <| (_ setEndpoint s3Endpoint)
 
-  def signUrl(bucket: Bucket, url: URI, expiration: DateTime): String = {
+  def signUrl(bucket: Bucket, url: URI, image: Image, expiration: DateTime): String = {
     // get path and remove leading `/`
     val key: Key = url.getPath.drop(1)
-    val request = new GeneratePresignedUrlRequest(bucket, key).withExpiration(expiration.toDate)
+
+    val filename: String = image.uploadInfo.filename match {
+      case Some(f) =>   s"${f.substring(0, f.lastIndexOf('.'))} (${image.id}).jpeg"
```

Comment:
There is technically no guarantee that there is a `.` in a filename. A RegEx might be safer here?